docs/patterns/triggers/triggers.mdx

```diff
@@ -157,4 +181,16 @@ toolset.triggers.subscribe(
 ```
 </Tab>
 </Tabs>
+Trigger event payload when the repository composio is starred:
+```json
+payload={
```

Comment:
Similar to the previous comment, the trigger event payload JSON could be improved:
1. Use double quotes for string values
2. Add proper indentation
3. Remove the 'payload=' prefix

Example:
```json
{
  "action": "created",
  "starred_at": "2024-12-11T15:31:26Z",
  "repository_id": 861033276,
  "repository_name": "composio",
  "repository_url": "https://github.com/composioHQ/composio/",
  "starred_by": "abhishekpatil4",
  "triggerName": "GITHUB_STAR_ADDED_EVENT"
}
```